deploy/charts/external-secrets/values.yaml

```diff
@@ -126,6 +126,27 @@ serviceMonitor:
   # -- Timeout if metrics can't be retrieved in given time interval
   scrapeTimeout: 25s
 
+  # -- Let prometheus add an exported_ prefix to conflicting labels
+  honorLabels: true
```

Comment:
I think this has been false prior this PR (this seems to be the default). Can we change this to `false`?

Reply:
fixed